platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -5,12 +5,145 @@
 #
 #-------------------------------------------------------------------------------
 
-set(NRF_COMMON_DIR ${CMAKE_CURRENT_LIST_DIR})
+set(NRF_COMMON_DIR "${CMAKE_CURRENT_LIST_DIR}")
 set(NRFX_DIR "${NRF_COMMON_DIR}/nrfx")
 
+#Specify the location of platform specific build dependencies.
+if(COMPILER STREQUAL "ARMCLANG")
+    set (S_SCATTER_FILE_NAME   "${PLATFORM_DIR}/common/armclang/tfm_common_s.sct")
+    set (BL2_SCATTER_FILE_NAME "${NRF_COMMON_DIR}/armclang/nordic_nrf_bl2.sct")
+    set (NS_SCATTER_FILE_NAME  "${NRF_COMMON_DIR}/armclang/nordic_nrf_ns.sct")
+    if (DEFINED CMSIS_5_DIR)
+      # not all project defines CMSIS_5_DIR, only the ones that use it.
+      set (RTX_LIB_PATH "${CMSIS_5_DIR}/CMSIS/RTOS2/RTX/Library/ARM/RTX_V8MBN.lib")
```

Comment:
This looks wrong @oyvindronningstad , should be MMN instead.
Not 100% sure, but i believe B stands for Baseline, M for Mainline
our SoCs are mainline ARMv8-M

Reply:
Thanks, good catch, I was not aware of the differences.